### ISSUE TITLE
GeoTIFF with different scale in x: 19.62030845109771 y: 19.620308451097525 is not currently supported

### ISSUE THREAD

**Issue by s9eenk** [IHCantabria/Leaflet.CanvasLayer.Field#10]
Facing an issue when rendering a GeoTIFF image. It prompts like "Make sure the difference is just a floating precision issue". I am confused.

Any help is appreciated . Thanks in advance !.

**Comment by VictorVelarde** [comment]
Hi Naveen, 

Thank you for your interest in the plugin. Currently it supports just regular grids, with square cells (same width-x and height-y). 

To render the cells in the right location the x-size is used now for both width and height. So if the difference between x and y is big, that could lead to misaligned cells. 

I've always used square cells, coming from WCS THREDDS, but I recently saw this potentially big issue, so I added the warning, as a quick and temporary solution. 

In your case, the difference is really so small 
19.62030845109771 vs 19.620308451097525 that I think it fits in the 'rounding category', so I would not really worry about it. 

Nevertheless, I'm gonna keep this issue open, as a reminder. 

Thank you!
